server/perfkit/explorer/handlers/base.py

```diff
@@ -90,6 +90,10 @@ def RenderHtml(self, template_file, template_values, status=200):
     template_values['static_dir'] = ('/_static/%s' %
                                      os.environ['CURRENT_VERSION_ID'])
     template_values['env'] = self.env
+    if 'PKE_DEBUG' in os.environ:
```

Comment:
done, add &debug=1 to the URL to activate it, or &debug=0 if you want to disable it while PKE_DEBUG is on.